workflow/routers.py

```diff
@@ -1,14 +1,136 @@
 # -*- coding: utf-8 -*-
-"""REST API URLs"""
+"""Workflow Routers"""
 
 
 from django.conf.urls import url, include
 from rest_framework.urlpatterns import format_suffix_patterns
 from rest_framework_nested import routers
+from collections import namedtuple
 
 from workflow import views
 
-workflow_router = routers.SimpleRouter(trailing_slash=False)
 
+Route = namedtuple('Route', ['url', 'mapping', 'name', 'initkwargs'])
+DynamicDetailRoute = namedtuple('DynamicDetailRoute', ['url', 'name', 'initkwargs'])
+DynamicListRoute = namedtuple('DynamicListRoute', ['url', 'name', 'initkwargs'])
+
+
+class NestedQueryableRecordRouter(routers.NestedDefaultRouter):
```

Comment:
Neet to modify the router to generate urls that work with queryRecord; so it only returns a single record rather than an array of records which queryRecord doesn't understand. The trailing slashes posed a problem here.